run.mac.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+osname=$(uname)
```

Comment:
@bbbales2, Try checking your aws s3 bucket at https://console.aws.amazon.com/s3/home?region=us-east-1 --- Can you access it? If not, there is some issue with granting S3 access to your creds